salt/cni/kube-flannel.yaml.jinja

```diff
@@ -46,6 +46,9 @@ data:
       {%- endif %}
       "Backend":
       {
+        {%- if salt.caasp_pillar.get('flannel:backend') in ["vxlan", "udp"] %}
```

Comment:
I think we should not force users to specify a port for flannel. I would rewrite this as {%- if salt.caasp_pillar.get('flannel:backend') in ["vxlan", "udp"] and salt.caasp_pillar.get('flannel:port') %}